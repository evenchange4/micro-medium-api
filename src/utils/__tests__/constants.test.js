const constants = require('../constants');

it('should return constants', () => {
  expect(constants).toMatchSnapshot();
});
