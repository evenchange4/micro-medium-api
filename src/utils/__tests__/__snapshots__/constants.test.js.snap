// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return constants 1`] = `
Object {
  "BASE_URL": "https://medium.com",
}
`;
